cdap-ui/app/cdap/components/PluginJSONCreator/Create/Content/ConfigurationGroupPage/GroupPanel/GroupActionButtons/index.tsx

```diff
@@ -40,10 +40,14 @@ const GroupActionButtonsView: React.FC<IGroupActionButtonsProps> = ({
 }) => {
   return (
     <div className={classes.groupActionButtons}>
-      <IconButton onClick={onAddConfigurationGroup}>
+      <IconButton onClick={onAddConfigurationGroup} data-cy="add-configuration-group">
```

Comment:
nit: I think we usually end the data-cy attribute of buttons with "-btn"

Reply:
Resolved